submission_runner.py

```diff
@@ -602,6 +605,9 @@ def main(_):
   if FLAGS.workload == 'librispeech_conformer':
     os.environ['XLA_PYTHON_CLIENT_MEM_FRACTION'] = '0.85'
 
+  if FLAGS.set_pytorch_max_split_size is True:
```

Comment:
Nit: Also, we can set `if FLAGS.set_pytorch_max_split_size: ...`.

Reply:
Fixed, thanks!